cpp/include/cudf/fixed_point/fixed_point.hpp

```diff
@@ -0,0 +1,276 @@
+/*
+ * Copyright (c) 2020, NVIDIA CORPORATION.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include <cmath>
+#include <cassert>
+#include <functional>
+#include <boost/serialization/strong_typedef.hpp>
```

Comment:
Can we avoid adding a boost dependency?

Reply:
FWIW we already depend on `arrow-cpp` which depends on `boost` so we already have a transitive dependency, but I agree I'd prefer to not have a direct dependency on `boost` if possible in libcudf.

Reply:
Boost dependency removed.